### PR TITLE
Various minor fixes

### PR DESCRIPTION
**Pull request by hellvinz**
Hi!
this PR aims at:

1. making the gem works as expected from a rubygem: since the gem is called 'influxdb-client', `require 'influxdb-client'` should be enough, `require 'influxdb2/client'` is counter intuitive
2. fixing documentation referencing InfluxDB, the namespace does not exists anymore

- [x] CHANGELOG.md updated
- [x] Rebased/mergeable
- [x] Tests pass
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+on: push
```
Comment (hellvinz) [inline .github/workflows/push.yml:1]: If you want to drop CircleCI. You can see an example of the results of the tests here https://github.com/hellvinz/influxdb-client-ruby/runs/661529686

**Comment by hellvinz** [comment]
> Hi @hellvinz, thanks for your PR.
> 
> Could you please also change an example: `examples/influxdb_18_example.rb` and update `CHANGELOG.md`?
> 
> Regards

sure! wondering if we should add ruby 1.8 to the testing matrix in circle CI?

**Comment by hellvinz** [comment]
Hi Jakub,
I have another naming question since:
- this gem is an iteration towards influxdb2 and the flux language
- this gem is meant to replace the `influxdb`gem
- there's an `InfluxDB2`namespace

I'm wondering wether it wouln't be the right time - before the influx2 release - to harmonize the naming and either:
- name this gem `influxdb2` and keep the `InfluxDB2` namespace
- keep the `influxdb-client` and change the namespace `InfluxDB2`to `InfluxDB`

**Comment by bednar** [comment]
I prefer way that keep the gem name to `influxdb-client`. The InfluxDB 1.8 introduces [forward compatibility](https://docs.influxdata.com/influxdb/v1.8/tools/api/#influxdb-2-0-api-compatibility-endpoints)... so this client is also suitable for 1.8+. 

The namespace `InfluxDB2` allows use old and new client together.

**Comment by bednar** [comment]
Hi @giovannelli, what do you think about this?

**Pull request merged**